### PR TITLE
README: pre-empt FAQ in usage section

### PR DESCRIPTION
**Pull request by pinheadmz**
Already getting new users reporting `"Error: Invalid checksum: proof.json"` in support channels.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,16 @@ mainnet merkle tree.
 
 ## Usage
 
+The airdrop tree was most recently updated on Dec 11, 2019.
```
Comment (boymanjor) [inline README.md]: @pinheadmz we'll need to update the date before we merge this.

Comment (pinheadmz) [reply README.md]: I think I got that date from commits in this repo but it could be wrong.

Comment (boymanjor) [reply README.md]: That date was correct when you opened this PR. The tree was updated at 9ab50eeb5b3f6ceb785cc94394facc6ea630096c.

Comment (pinheadmz) [reply README.md]: updated

**Comment by boymanjor** [comment]
LGTM

**Pull request merged**